index.html

```diff
@@ -306,12 +304,14 @@ <h2><code>PointerEvent</code> Interface</h2>
     readonly        attribute long        tiltX;
     readonly        attribute long        tiltY;
     readonly        attribute long        twist;
-    readonly        attribute double      altitudeAngle;
-    readonly        attribute double      azimuthAngle;
     readonly        attribute DOMString   pointerType;
     readonly        attribute boolean     isPrimary;
     [SecureContext] sequence&lt;PointerEvent> getCoalescedEvents();
     sequence&lt;PointerEvent> getPredictedEvents();
+    static                    double      getAzimuthAngle(long tiltX, long tiltY);
+    static                    double      getAltitudeAngle(long tiltX, long tiltY);
+    static                    long        getTiltX(double azimuthAngle, double altitudeAngle);
+    static                    long        getTiltY(double azimuthAngle, double altitudeAngle);
```

Comment:
Bikeshed: I'm not sure the "get" prefix makes much sense here, since this isn't getting some pre-stored data. I'd just name them `azimuthAngle(...)` etc.

Reply:
@domenic does web idl support overloading for attributes? If I use `static long tiltX(double azimuthAngle, double altitudeAngle)` will it work given there is a `readonly attribute long tiltX`?